actions/custom.js

```diff
@@ -37,48 +29,37 @@ export const formatDateFromMillis = dateMillis => {
 /**
  * Format date from backend date string.
  */
-export const formatDate = (date) => {
+export const formatDate = (date, timezone) => {
```

Comment:
Just out of curiosity, what was the thought process behind passing timezone to every format* function, as opposed to calling TimeZoneService.getTimeZone() directly inside the format function or similar?

Reply:
I try to import timezone in custom.js, but I found it's must be default value because ./action not below App.js.
Or, think can change thought. I will try to place timezone in other way.